frontend/plan/components/schedule/ScheduleSelectorDropdown.tsx

```diff
@@ -196,7 +195,8 @@ const DropdownButton = ({
             >
                 <i className="far fa-copy" aria-hidden="true" />
             </Icon>
-            <Icon
+            {/* TODO: Add back when working */}
```

Comment:
What is this for?

Reply:
This is for exporting PCP schedule to an external calendar app. This feature doesnt work currently so I'm going to remove it temporary on the frontend side (Alex is working on fixing this)